### ISSUE TITLE
Remove yarn requirement

### ISSUE THREAD

**Issue by forgetso** [polkadot-js/dev#1122]
It states in this package that 

```
Currently only
        yarn supports package.json workspaces, hence the limitation.
```

This is now false as npm has very reasonable workspaces support. This check prevents local installation of polkadotJS packages when using npm workspaces.

e.g 

- polkadotJS is in `/usr/pjs` and has been built using `yarn build`
- npm workspace is in `/usr/wrk`
- polkadotJS deps are referenced in `/usr/wrk` package.json like `"@polkadot/api: file:../pjs"`

**Comment by jacogr** [comment]
I guess the reasoning behind this has certainly changed since it has been added - but at this point the dev scripts are tied to the hip with yarn. (As are the dependencies, i.e. the actual yarn berry binary is part of the repo)

So the error can certainly be adjusted, but the requirement is not going away - this only support yarn as part of the CI, local build & management processes.

Some links -

- [yarn binaries](https://github.com/polkadot-js/dev/tree/master/.yarn/releases)
- [yarn usage](https://github.com/polkadot-js/dev/blob/master/packages/dev/scripts/polkadot-dev-build-ts.mjs#L48) (there are a lot more, also in the repo management scripts)
- [yarn lockfiles only](https://github.com/polkadot-js/dev/blob/master/yarn.lock)

Nothing of this is insurmountable - as with everything it is just a question of time and effort.

TL;DR The requirement (without a lot of effort and adjustments) needs to stay at this point - however the error messaging (while it was correct) is certainly misleading at this point, but PRs to adjust would be appreciated if it is a concern to be completely misleading.
